script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -38,7 +38,7 @@ There's no way to get out of this without telling her everything.
 <Page12>
 "I don't how to explain it... but I can see lines in things that can be cut."
 "Lines that can be cut?"
-I know, right? Normally people wouldn't believe me when I tell them this.
+I know, right? Normally, people wouldn't believe me if I told them.
```

Comment:
Normally, people don't believe me when I tell them.

Reply:
I think the intent during the session was that he _hasn't_ told people (so they _wouldn't_ even if he did tell them), but he _did_ tell the doctors etc. 7 years ago. He probably doesn't tell people anymore though, so I think wouldn't is still usable.

Reply:
Maybe `wouldn't believe me even if I told them` makes it more clear that that's the phrasing intent.